src/components/Layout/AuthLayout/AuthLayout.tsx

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import { Outlet } from 'react-router-dom';
-import AppBar from '@mui/material/AppBar';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
 import Box from '@mui/material/Box';
-
+const styles = {
+	display: 'flex',
+	justifyContent: 'space-between',
+	height: '65px',
+	backgroundColor: '#3d7cb9',
+};
```

Comment:
It must be in a .styles.ts file. Example: AuthLayout.styles.tsx

We try to avoid mixing ts/js with css in all possible cases.

```suggestion
const container= {
	display: 'flex',
	justifyContent: 'space-between',
	height: '65px',
	backgroundColor: '#3d7cb9',
};

const styles = { container };
export default styles;
```